src/arena/ArenaStats.tsx

```diff
@@ -0,0 +1,86 @@
+import { I18n } from "@aws-amplify/core";
+
+import StatsBlock from "../components/StatsBlock";
+import { GenMission } from "./interface";
+
+interface ActivePersonMission {
+  dailyCompletion: number;
+  dailyScore: number;
+  missions: GenMission[];
```

Comment:
You are doing a great job typing everything out - this is INCREDIBLY appreciated!